cmd/libs/go2idl/generator/default_generator.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package generator
+
+import (
+	"io"
+
+	"k8s.io/kubernetes/cmd/libs/go2idl/namer"
+	"k8s.io/kubernetes/cmd/libs/go2idl/types"
+)
+
+// DefaultGen implements a do-nothing Generator.
+//
+// It can be used to implement static content files.
+type DefaultGen struct {
+	// OptionalName is optional, if present it will be used for the generator's
+	// name, and the filename (with ".go" appended).
+	OptionalName string
```

Comment:
Sorry - I didn't notice the conflict with Name() method.
I think that having "OptionalName" and "Body" (which is optional) is much more misleading.
Since I don't have better suggestion here, I think that getting back to OptionalBody is better choice.
WDYT?